index.js

```diff
@@ -2613,6 +2613,42 @@
   }
   S.ifElse = def('ifElse', {}, [Pred(a), Fn(a, b), Fn(a, b), a, b], ifElse);
 
+  //# when :: (a -> Boolean) -> (a -> b) -> a -> b
+  //.
+  //. Takes a unary predicate, a unary function, and a value of any type, and
+  //. returns the result of applying the function to the value if the value
+  //. satisfies the predicate; the value otherwise.
+  //.
+  //. ```javascript
+  //. > S.when(x => x < 0, Math.abs, -1)
+  //. 1
+  //.
+  //. > S.when(x => x < 0, Math.abs, 4)
+  //. 4
+  //. ```
```

Comment:
The problem with these examples is that `Math.abs(x)` is more direct than `S.when(x => x < 0, Math.abs, x)`. Perhaps we should mirror the `unless` examples:

```javascript
> S.when(x => x >= 0, Math.sqrt, 16)
4

> S.when(x => x >= 0, Math.sqrt, -1)
-1
```